protected/humhub/modules/notification/migrations/m240422_162959_new_is_untouched_settings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use humhub\modules\notification\components\NotificationManager;
+use humhub\modules\notification\Module;
+use humhub\modules\user\models\User;
+use yii\db\Migration;
+
+/**
+ * Class m240422_162959_new_is_untouched_settings
+ */
+class m240422_162959_new_is_untouched_settings extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        foreach (User::find()->each() as $user) {
```

Comment:
If possible, we should work directly on the database.

Installations with many users could otherwise have problems due to many queries to the database. 

Is this query complete?
```sql
INSERT INTO contentcontainer_settings (module_id, contentcontainer_id, name, value)  
SELECT module_id, contentcontainer_id, "is_touched_settings" as name, "1" as value  
FROM contentcontainer_settings
WHERE name="notification.like_email" and module_id="notification";
```

Reply:
@luke- Thanks very much for the SQL command!

Added to commit https://github.com/humhub/humhub/pull/6964/commits/b115707eb036d22f29e839c4217a514d35697b45 and tested